src/ontology/mondo-ingest.Makefile

```diff
@@ -181,7 +181,11 @@ config/%_term_exclusions.txt config/%_exclusion_reasons.robot.template.tsv: conf
 	--component-signature-path $(word 4,$^) \
 	--config-path $(word 5,$^)
 
-all-exclusions: config/ordo_term_exclusions.txt config/doid_term_exclusions.txt config/omim_term_exclusions.txt config/icd10cm_term_exclusions.txt config/icd10who_term_exclusions.txt config/ncit_term_exclusions.txt
+# Combines all tables
+config/term_exclusions.txt config/exclusion_reasons.robot.template.tsv: config/ordo_term_exclusions.txt config/doid_term_exclusions.txt config/omim_term_exclusions.txt config/icd10cm_term_exclusions.txt config/icd10who_term_exclusions.txt config/ncit_term_exclusions.txt
```

Comment:
This is ready to merge if we want the TSV / TXT stuff now.

I'm unclear on (i) docs, and (ii) OBO/TTV. I put my questions here: https://github.com/monarch-initiative/mondo-ingest/issues/54